app/services/loader.js

```diff
@@ -13,7 +13,7 @@ export default Service.extend({
     if (config.isExternal) {
       options.url = url;
     } else {
-      options.url = config.withoutPrefix ? `${adapter.host}/${url}` : `${adapter.urlPrefix()}${url}`;
+      options.url = config.withoutPrefix ? `${adapter.host}${url}` : `${adapter.urlPrefix()}${url}`;
```

Comment:
At the last? Leading slash means at the start. 

I think you're completely misunderstanding this. I'll get home and explain with a code sample.